tests/accordion-test.ts

```diff
@@ -0,0 +1,414 @@
+            </aalam-accordion>
+        `);
+        expect(el.querySelectorAll(".acc-body.acc-active").length).to.equal(1);
+        expect(el.querySelector(".acc-body.acc-active")).to.exist;
+
+        const element = await fixture(html`
+            <aalam-accordion>
+                <div class="container acc-active">
+                    <div class="acc-title">Title 1</div>
+                    <div class="acc-body acc-active">Content 1</div>
+                </div>
+                <div class="container acc-active">
+                    <div class="acc-title">Title 2</div>
+                    <div class="acc-body acc-active">Content 2</div>
+                </div>
+            </aalam-accordion>
+        `);
+        expect(
+            element.querySelectorAll(".acc-body.acc-active").length
+        ).to.equal(2);
+        expect(element.querySelector(".acc-body.acc-active")).to.exist;
+        expect(element.querySelectorAll(".acc-active")).to.exist;
+    });
+
+    it("allows multiple items to be open if nomultiple y ", async () => {
+        const el = await fixture(html`
+            <aalam-accordion activecls="acc-active">
+                <div class="container ">
+                    <div class="acc-title">
+                        <span>What is Artificial Intelligence?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">
+                        <p>body 1</p>
+                    </div>
+                </div>
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>What is Machine Learning?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 2</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 3?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 3</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 4</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 4</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 5</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 5</div>
+                </div>
+            </aalam-accordion>
+        `);
+
+        const titles = el.querySelectorAll(".acc-title");
+        titles[0]?.click();
+        titles[1]?.click();
+        await el.updateComplete;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(2);
```

Comment:
Check acc-active alone doesnt imply if the accordion is really open. Its not safe to assume that the application would handle it correctly with this class. Its better to make sure the body's display is set to open for sure.